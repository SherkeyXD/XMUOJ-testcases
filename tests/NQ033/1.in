2
13159269056
13159234089