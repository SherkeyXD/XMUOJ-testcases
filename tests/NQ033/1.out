669056
634089
